pkg/util/podname/namer_test.go

```diff
@@ -0,0 +1,36 @@
+package podname
+
+import (
+	"testing"
+)
+
+const longName = "really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-long-name"
+
+func TestGetName(t *testing.T) {
```

Comment:
May not matter right now, but you might have been able to do this algorithmically generating random chars from 1 to util.DNS1123SubdomainMaxLength+(something), and also adding increasing length random suffixes. Might work better to catch edge cases.